packages/data-api/src/TupaiaDataApi.js

```diff
@@ -179,6 +179,11 @@ export class TupaiaDataApi {
 
   buildOptionsMetadata = (options = [], type) => {
     const optionList = options.length === 0 && type === 'Binary' ? DEFAULT_BINARY_OPTIONS : options;
+
+    if (!optionList.length) {
+      return undefined;
+    }
+
```

Comment:
Just to fix this bug here (https://github.com/beyondessential/tupaia-backlog/issues/1322#issuecomment-777075175). This is because in `tableOfDataValues` data builder, it doesn't check if `options` metadata is empty or not to get the value (below)
![options](https://user-images.githubusercontent.com/63621318/107589283-c486f480-6c59-11eb-9310-b3720f31f810.PNG)

So I just return undefined here if there are no options, I think this also aligns with how DHIS2 return options metadata